### DISCUSSION TITLE
Support for BentoML Docker image running in GCP Cloud Run

### DISCUSSION THREAD

**Discussion by Sujith93** [bentoml/BentoML#994] [General]
Hi,
For building the docker image I took steps mentioned below link as a reference.
[https://docs.bentoml.org/en/latest/deployment/google_cloud_run.html#create-google-cloud-project](url)
But instead of `git clone git@github.com:bentoml/BentoML.git`
I build the model in the local with the iris dataset and also build a docker image.
By doing docker run, I confirm the docker image is working fine also got the predictions in swagger UI
So, Next, I followed the same steps to push the image into the cloud repository and then to Cloud Run.
But will creating cloud run I got the bellow error

![image](https://user-images.githubusercontent.com/15892407/90246647-b135d080-de52-11ea-9e9b-1c759c6a3d9a.png)

From the logs its shows 
`ModuleNotFoundError: No module named bentoml`
![image](https://user-images.githubusercontent.com/15892407/90246778-f78b2f80-de52-11ea-8735-e0e3cefb8b33.png)

Everything is working fine in local, but when I pushed to Cloud Run, it throws an error.
Could you please help me out,
Thanks in advance !!

**Comment by parano** [comment]
Hi @Sujith93  - when you verify the docker image locally, did you use the docker image built with `gcloud builds submit` CLI command? or did you use `docker build` to create a test image?

**Reply by Sujith93** [reply]
In local, I went to the saved path and executed the below commands,
_For docker build_
`sudo docker build -t iris_model .`
_For running the image_
`sudo docker run -it -d -p 5000:5000  iris_model`

_For Pushing local image into the container registry_
`gcloud builds submit --tag gcr.io/project-name/iris_model`

_Also tried pushing the image directly to container registry_
`sudo docker tag iris_model gcr.io/project-name/iris_model`
`sudo gcloud docker -- push  gcr.io/project-name/iris_model`

But the issues still remain the same.

I also verified by building a docker image with a flask app (without using BentoML) and pushed it to container registry and it is working fine.

Could you please help me out, why this issue is coming for me and where I was mistaken

**Reply by parano** [reply]
Hi @Sujith93, this actually looks like an issue on the google cloud site, have you tried reaching out to their support team?
